ros_buildfarm/templates/snippet/rosdep_init.Dockerfile.em

```diff
@@ -0,0 +1,8 @@
+RUN mkdir -p /etc/ros/rosdep/sources.list.d
+
+@[for url in custom_rosdep_urls]@
+RUN (cd /etc/ros/rosdep/sources.list.d/ && wget --progress=dot @url )
+@[end for]@
+@[if not custom_rosdep_urls]@
+RUN rosdep init
```

Comment:
Ok, I'll update that section to unconditionally run rosdep init.
I would prefer to keep the ability to override the default sources. Specifically I would like to fetch all of my rosdep sources from my forked rosdistro repo (whether or not they are identical to the public one). The primary motivation for this is that my rosdistro repo is hosted on an internal git server rather than GitHub, and so it's insulated from issues with GitHub or Internet connectivity at the site.

I can update the docs so it's clear to users that they can either use a combination of the default sources and their own custom sources file, or avoid the default sources entirely and link to the file in their forked rosdistro repo.